plugins/scanners/scancode/src/main/kotlin/ScanCode.kt

```diff
@@ -135,8 +135,8 @@ class ScanCode internal constructor(
     override fun transformVersion(output: String): String {
         // On first use, the output is prefixed by "Configuring ScanCode for first use...". The version string can be
         // something like:
-        // ScanCode version 2.0.1.post1.fb67a181
-        // ScanCode version: 31.0.0b4
+        // ScanCode version 30.0.0
+        // ScanCode version: 31.0.0
```

Comment:
The entire clarifications could be removed, because this is nicely documented by the unit test for this function.

Reply:
I actually wasn't sure whether I really copied the console output as-is back then, or whether the data is synthetic, so went for the safe approach and looked it up again.

Reply:
But you're probably right, now that I verified the correctness of the data, I should probably remove this redundancy. I'll amend again.

Reply:
Feel free to do it in a separate PR if that makes sense.